### PR TITLE
Updated Component Test Examples

### PR DESCRIPTION
**Pull request by davideziviani**
Updated th example according to the changes in ACHP

### CODE REVIEW

File: ComponentTests/__init__.py [hunk 1]
```diff
@@ -0,0 +1,5 @@
+from Condenser import CondenserClass
```
Comment (abahman) [inline ComponentTests/__init__.py]: @davideziviani Do you think we need this? 
I think we can also remove it from the main source code.

File: ComponentTests/CompressorTest.py [hunk 2]
```diff
@@ -1,26 +1,34 @@
...
 
+Ref = 'R134a'
+Backend = 'HEOS' #choose between: 'HEOS','TTSE&HEOS','BICUBIC&HEOS','REFPROP','SRK','PR'
+AS = CP.AbstractState(Backend, Ref)
+    
 kwds={
       'M':[217.3163128,5.094492028,-0.593170311,4.38E-02,-2.14E-02,1.04E-02,7.90E-05,-5.73E-05,1.79E-04,-8.08E-05],
       'P':[-561.3615705,-15.62601841,46.92506685,-0.217949552,0.435062616,-0.442400826,2.25E-04,2.37E-03,-3.32E-03,2.50E-03],
-      'Ref':'R134a',
+      'Ref':Ref,
```
Comment (ibell) [inline ComponentTests/CompressorTest.py]: This still needed now that you use AS?

File: ComponentTests/CompressorTest.py [hunk 3]
```diff
@@ -1,26 +1,34 @@
-from Compressor import CompressorClass
+from ACHP.Compressor import CompressorClass
```
Comment (ibell) [inline ComponentTests/CompressorTest.py]: You also need a from __future__ import print_function for python 2/3 support

File: ComponentTests/MCETest.py [hunk 4]
```diff
@@ -29,67 +31,67 @@
...
+        'hin_r': PropsSI('H','P',PropsSI('P','T',282.0,'Q',1.0,Ref),'Q',0.15,Ref),
         'Verbosity':0,
-        'Backend':'HEOS' #choose between: 'HEOS','TTSE&HEOS','BICUBIC&HEOS','REFPROP','SRK','PR'
         }
 MCE=MultiCircuitEvaporatorClass(**kwargs)
 MCE.Calculate()
-print 'MCE Q='+str(MCE.Q)+' W w/o mal-distribution' 
+print ('MCE Q='+str(MCE.Q)+' W w/o mal-distribution')
 
 #Not exactly the same since
```
Comment (ibell) [inline ComponentTests/MCETest.py]: Inconsistent spacing in comments

File: ComponentTests/CompressorTest.py (hunk 2 above)
Comment (abahman) [reply ComponentTests/CompressorTest.py]: it is used to determine the solubility of refrigerant in oil.

**Pull request merged**